### PR TITLE
chore: Taxjar signup button

### PR DESCRIPTION
**Pull request by shantanu-kadam-epi**
Asana link : https://app.asana.com/0/1200595970312551/1200571214737045

image

### CODE REVIEW

File: erpnext/erpnext_integrations/doctype/taxjar_settings/taxjar_settings.js [hunk 1]
```diff
@@ -7,3 +7,8 @@ frappe.ui.form.on('TaxJar Settings', {
 		frm.toggle_reqd("sandbox_api_key", frm.doc.is_sandbox);
 	}
 });
+frappe.ui.form.on("TaxJar Settings", "refresh", function(frm){
+    frm.add_custom_button("Sign Up For Taxjar", function(){
+			var myWin = window.open('https://app.taxjar.com/api_sign_up');
```
Comment (vishdha) [inline erpnext/erpnext_integrations/doctype/taxjar_settings/taxjar_settings.js]:
```suggestion
			window.open('https://app.taxjar.com/api_sign_up');
```

**Review by vishdha — CHANGES_REQUESTED** [review]

File: erpnext/erpnext_integrations/doctype/taxjar_settings/taxjar_settings.js [hunk 2]
```diff
@@ -7,3 +7,8 @@ frappe.ui.form.on('TaxJar Settings', {
 		frm.toggle_reqd("sandbox_api_key", frm.doc.is_sandbox);
 	}
 });
+frappe.ui.form.on("TaxJar Settings", "refresh", function(frm){
+    frm.add_custom_button("Sign Up For Taxjar", function(){
```
Comment (vishdha) [inline erpnext/erpnext_integrations/doctype/taxjar_settings/taxjar_settings.js]:
```suggestion
frappe.ui.form.on("TaxJar Settings", "refresh", function(frm) {
    frm.add_custom_button("Sign Up For Taxjar", function() {
```

**Review by vishdha — CHANGES_REQUESTED** [review]

**Review by vishdha — APPROVED** [review]

**Comment by sahil28297** [comment]
@vishdha why are we adding a signup button in our product?

**Comment by vishdha** [comment]
@sahil28297 https://developers.taxjar.com/integrations/certification-checklist/

**Review by sahil28297 — APPROVED** [review]

**Pull request merged**